templates/taxi/car_confirm_delete.html

```diff
@@ -1,10 +1,17 @@
 {% extends "base.html" %}
 
 {% block content %}
-  <h1>Delete car?</h1>
-  <form action="" method="post">
-    {% csrf_token %}
+  <h1>Delete car {{ car.model }}?</h1>
+    <div class="row">
+      <div class="col-1">
+        <a href="{% url 'taxi:car-detail' car.id %}"><input type="submit" value="Cancel" class="btn btn-secondary"></a>
```

Comment:
The 'Cancel' button is implemented as a link wrapped around a submit button, which is not semantically correct. Consider using a regular anchor tag styled as a button for the 'Cancel' action, like this: `<a href="{% url 'taxi:car-detail' car.id %}" class="btn btn-secondary">Cancel</a>`.

Reply:
changed